src/modules/list.marko

```diff
@@ -0,0 +1,359 @@
+                                Item 1 with an action
+                            </span>
+                            <span class="list__trailing">
+                                <span class="switch">
+                                    <span
+                                        aria-labelledby="switch-in-list"
+                                        class="switch__control"
+                                        role="switch"
+                                        tabindex="0"
+                                        aria-checked="false"
+                                    />
+                                    <span class="switch__button"/>
+                                </span>
+
+                            </span>
+                        </div>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </div>
+    <highlight-code type="html" >
+            <div class="list">
+            <ul>
+                <li>
+                    <button>
+                        Button
+                    </button>
+                </li>
+                <li>
+                    <a href="www.ebay.com">
+                        <span class="list__body">
+                            Link
+                        </span>
+                        <span class="list__trailing">
+                            <svg aria-hidden="true" class="icon icon--16" height="16" width="16">
+                                <use href="#icon-chevron-right-16"/>
+                            </svg>
+                        </span>
+                    </a>
+                </li>
+                <li>
+                    <div>
+                        <span class="list__body" id="switch-in-list">
+                            Item 1 with an action
+                        </span>
+                        <span class="list__trailing">
+                            <span class="switch">
+                                <span
+                                    aria-labelledby="switch-in-list"
+                                    class="switch__control"
+                                    role="switch"
+                                    tabindex="0"
+                                    aria-checked="false"
+                                />
+                                <span class="switch__button"/>
+                            </span>
+
+                        </span>
+                    </div>
+                </li>
+            </ul>
+        </div>
+    </highlight-code>
+
+    <h3 id="list-with-divider">List with dividers</h3>
+    <p> To add a divider, add a <span class="highlight">hr</span> element after the list item. </p>
+   <div class="demo">
+        <div class="demo__inner">
+            <div class="list">
+                <ul>
+                    <li>
+                        <div>
+                            <span class="list__leading">
```

Comment:
I don't love `leading` as a name, I think it's the wrong... tense? Not sure, but something feels off about it.

Reply:
List__lead 
List__start
List__leader

For the last one
List__end
List__trail
List__trailer

Im thinking the start and end make the most sense since they're flex terms.